Phraends_Flask/Backend/Crawler/Crawler.py

```diff
@@ -178,3 +178,69 @@ def get_news_from_dow_jones(ticker):  # apple
         open("the_news_texts.txt", mode="r"),
     )  # type .read() can read the content
 
+
+def get_news_from_cnbc(ticker):
+    """
+    Summary: 
+        Search the ticker name in cnbc, and then crawl down the first five articles.
+    
+    Args:
+        ticker (string): the ticker name of the stock
+    
+    Returns:
+        links (list): the links to the five articles
+        open("the_news_texts.txt", mode="r") (txt file): the txt file contains the five articles
+    """
+    driver = webdriver.Chrome()
+    driver.get("https://www.cnbc.com/")
+
+    # Click the search button
+    search_point = driver.find_element(By.CLASS_NAME, "icon-search")
+    search_point.click()
+
+    # Send in the ticker name
+    search = driver.find_element(
+        By.XPATH, '/html/body/div[3]/div/div[1]/header/div[4]/div/div/div[3]/section/div[2]/div[1]/section/form/input[2]'
+    )
+    search.send_keys(str(ticker))
+    search.send_keys(Keys.RETURN)
+
+    # Collect the 5 latest news content
+    links = []
+    driver.implicitly_wait(5)
+    f = open("the_news_texts.txt", "w")
+    for i in range(0, 5):

```

Comment:
Could you modified it and make it returning 5 latest articles that is availabe? I believe using while loop will be an easy implementation.